manager/integration/tests/test_basic.py

```diff
@@ -280,6 +280,11 @@ def backup_test(client, host_id, volname):
     setting = client.update(setting, value=common.get_backupstore_url())
     assert setting["value"] == common.get_backupstore_url()
 
+    credential = client.by_id_setting("backupTargetCredentialSecret")
```

Comment:
Only if the backuptarget is `s3://`

You can double check by switch the backupstore to `nfs`